pyproject.toml

```diff
@@ -160,12 +160,14 @@ ignore = [
   "DTZ005",  # TODO: Fix these
   "B904",  # TODO: Fix these
   "A001",  # TODO: Fix these
+  "A004",  # TODO: Fix these
   "B006",  # TODO: Fix these
   "S607",  # TODO: Fix these
   "S108",  # TODO: Fix these
   "S602",  # TODO: Fix these
   "E402",  # TODO: Fix these
   "UP028",  # TODO: Fix these
+  "UP031",  # TODO: Fix these
   "BLE001",  # TODO: Fix these
```

Comment:
I don't know why new style issues are suddenly cropping up. It looks like ruff was just missing them before. My local version isn't catching them either.